Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.List.ts2

```diff
@@ -45,4 +50,14 @@ prototype(Neos.NeosIo:Reference.List.Item) < prototype(TYPO3.TypoScript:Template
 		}
 	}
 	projectType = ${q(q(node).property('projectType')).property('title')}
+
+	@cache {
```

Comment:
From my experience this is faster in cases where caching every result
variant is too complicated.